tests/integration/benchmark_runner/common/oc/test_oc.py

```diff
@@ -66,7 +66,17 @@ def test_oc_get_odf_version():
     """
     oc = OC(kubeadmin_password=test_environment_variable['kubeadmin_password'])
     oc.login()
-    oc.get_odf_version()
+    assert oc.get_odf_version()
+
+
+def test_oc_get_odf_disk_count():
+    """
+    This method get odf disk count
+    :return:
+    """
+    oc = OC(kubeadmin_password=test_environment_variable['kubeadmin_password'])
+    oc.login()
+    assert oc.get_odf_disk_count() > 0
```

Comment:
Make sure that this works correctly if ODF is not installed.

Reply:
We must have ODF on our CI env

Reply:
Our CI environment, yes, but this is a public repo, and someone may want to run without ODF.

Reply:
Yes, you right. fixed

Reply:
This will still fail, because get_odf_disk_count won't return anything.  If you use my suggested fix above, it will be something like:

```
odf_disk_count = oc.get_odf_disk_count()
assert(odf_disk_count is None or odf_disk_count > 0)
```